storage/paths/db_index.go

```diff
@@ -120,6 +127,8 @@ func splitString(str string) []string {
 }
 
 func (dbi *DBIndex) StorageAttach(ctx context.Context, si storiface.StorageInfo, st fsutil.FsStat) error {
+	ctx = addNetworkTag(ctx)
```

Comment:
I'm still left with the question of where this context is coming from and whether we can fix it closer to creation